src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/FindAllReferencesHandler.cs

```diff
@@ -137,7 +204,18 @@ public async Task<VSReferenceItem[]> HandleRequestAsync(ReferenceParams request,
                 remappedLocations.Add(referenceItem);
             }
 
-            return remappedLocations.ToArray();
+            progress.Report(remappedLocations.ToArray());
         }
     }
+
+    // Temporary while the PartialResultToken serialization fix is in
+    [DataContract]
+    public class CustomReferenceParams : TextDocumentPositionParams
```

Comment:
I'd throw this into the class above it and make it private

```suggestion
    private class SerializableReferenceParams : TextDocumentPositionParams
```

Reply:
Sure, note this is just (extremely) temporary till we get the serialization bug fix in.